chapters/ch05.asciidoc

```diff
@@ -136,6 +138,8 @@ This allows us to create slightly modified shallow copies of other objects. When
 
 Another benefit worth pointing out is that `Object.assign()` can cause accidents: if we forget to pass an empty object literal as the first argument for this use case, we end up mutating the object. With object spread, there is no way to accidentally mutate anything, since the pattern always acts as if an empty object was passed to `Object.assign` in the first position.
 
+> Some of the above phrasing matches [our I/O 2018 talk](https://www.youtube.com/watch?v=mIWCLOftfRw) *exactly*! I love it.
```

Comment:
Comes straight from your own comment here: https://github.com/mjavascript/mastering-modular-javascript/pull/4/files#diff-4271760ad3c0338ba0897714ab786a23R92 😅

Reply:
Nice to see you love your own wording! Hahaha. I had the same thing happen to me the other day, where I was like "this is a fantastic combination of words", and like 5 minutes later realized I had written it 😂